tdvt/tdvt/tdvt.py

```diff
@@ -510,9 +510,11 @@ def run_tests_impl(tests, max_threads, args):
     logging.debug("test queue size is: " + str(len(all_work)))
 
     if not smoke_tests:
-        logging.warning("""No smoke tests detected. Tests will attempt to run without first verifying the data source connection. This may result in tests failing because of connection, rather than plugin, issues.""")
         if args.smoke_test:
+            logging.warning("No smoke tests detected. Check your data source's .ini file to verify that at least one test has SmokeTest = True set.")  # noqa: E501
             sys.exit(1)
+        else:
+            logging.warning("No smoke tests detected. Tests will attempt to run without first verifying the data source connection. This may result in tests failing because of connection, rather than plugin, issues.")  # noqa: E501
```

Comment:
"This may result in tests failing because of connection, rather than plugin, issues."

While you're editing the file, can you edit this log message to not be plugin-specific, since TDVT is run against the old-style connectors internally? Maybe something like:
"This may result in test failures due to database connection issues rather than connector issues."